packages/contracts/src/contract-deployment/config.ts

```diff
@@ -230,10 +225,7 @@ export const makeContractDeployConfig = async (
     },
     OVM_ETH: {
       factory: getContractFactory('OVM_ETH'),
-      params: [
-        predeploys.OVM_L2CrossDomainMessenger,
-        '0x0000000000000000000000000000000000000000', // will be overridden by geth when state dump is ingested.  Storage key: 0x0000000000000000000000000000000000000000000000000000000000000008
-      ],
+      params: [], // need to set the owner to the L2 Bridge address in genesis.go
```

Comment:
What does this comment mean?

Reply:
The intent is similar to the comment that was removed on the left: 
(`// will be overridden by geth when state dump is ingested. Storage key: 0x0000000000000000000000000000000000000000000000000000000000000008`)
And is referring to the code here: 
https://github.com/ethereum-optimism/optimism/pull/988/files#diff-17b617e193a2e9627cf0c5459c05a94f7de9ce2ebde734ede16251f1a7a16118R313-R315